files/scripts/sonic-netns-exec

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Wrapper to execute any command in a specific
+# network namespace.
+# Usage:
+# sonic-netns-exec <namespace name> <command to be executed>
+NS="$1"
+shift
+if [ "$NS" != "" ]; then
+        ip netns exec $NS "$@"
```

Comment:
>"$@" [](start = 26, length = 4)

Did you test multiple argument such as `ls a.bin`? I guess you don't need quoting.

Reply:
yes, tested this out, removed quotes